src/OceanSeaIceModels/PrescribedAtmospheres.jl

```diff
@@ -318,6 +321,36 @@ function Base.show(io::IO, pa::PrescribedAtmosphere)
     print(io, "└── boundary_layer_height: ", prettysummary(pa.boundary_layer_height))
 end
 
+# set the clock to be the same as the ocean model
+function set!(model::PrescribedAtmosphere, checkpoint_file_path)
+    addr = checkpointer_address(model)
+
+    jldopen(checkpoint_file_path, "r") do file
+        checkpointed_clock = file["$addr/clock"]
+
+        # Update model clock
+        set_clock!(model, checkpointed_clock)
+    end
+
+    return nothing
+end
+
+checkpointer_address(::PrescribedAtmosphere) = "HydrostaticFreeSurfaceModel"
```

Comment:
Ideally we'd like to make the checkpointer also save the clock for the atmosphere and all other models and then read it everything off from the corresponding checkpointer address.

Reply:
I think that's for a future PR..

Reply:
I don't know, I'm apprehensive... I hate this
```
checkpointer_address(::PrescribedAtmosphere) = "HydrostaticFreeSurfaceModel"
```
 because it's a hack. But on the other hand I like the symmetry here:

https://github.com/CliMA/ClimaOcean.jl/blob/9f9ca89561211d535926dd4570d362b677eab6dc/src/OceanSeaIceModels/ocean_sea_ice_model.jl#L98-L108

Essentially the hack tells the checkpointer for the atmosphere that it should go read data from the ocean model checkpointer.

Reply:
I guess this sets the basis for future code development so in that sense is OK?

Reply:
why do you have to write 

```julia
checkpointer_address(::PrescribedAtmosphere) = "HydrostaticFreeSurfaceModel"
```

Reply:
I’m changing that
That’s part of the hack I was explaining yesterday I don’t like.